pmultiqc/modules/fragpipe/fragpipe.py

```
@@ -122,12 +158,107 @@ def get_data(self):
+
+        # Parse combined_ion.tsv for MS/MS counts
+        if self.fragpipe_files.get("combined_ion"):
+            try:
+                combined_ion_path = self.fragpipe_files["combined_ion"][0]
+                combined_ion_df, combined_ion_column_valid = combined_ion_reader(combined_ion_path)
+                log.info("Combined ion data loaded successfully.")
+
+                if combined_ion_df is not None and combined_ion_column_valid:
+                    self.msms_counts = get_msms_counts_per_peak(combined_ion_df)
+                    log.info("MS/MS counts per peak loaded successfully.")
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)